js/main.js

```diff
@@ -23,4 +23,6 @@ function submitAction(event) {
   data.entries.unshift(inputs);
   imageChange.setAttribute('src', 'images/placeholder-image-square.jpg');
   formField.reset();
+  location.reload();
+  window.location.href = '#entries';
```

Comment:
We're going to avoid `location.reload()` for this app. Instead you will need to update your `data-view` divs so that the entry list is showing after the user submits the form. You'll be using your `addJournal` function here.